Open311_ServiceRequest/src/open311_ServiceRequest.gs

```diff
@@ -0,0 +1,67 @@
+function onEditIssues() {
+  var sheet = SpreadsheetApp.getActive().getSheetByName('Issues');
```

Comment:
I think the sheet name should define the type of standard data model. (e.g. Open311ServiceRequest)